index.html

```diff
@@ -122,6 +122,40 @@ <h2>A project for first-time contributions</h2>
         <!-- Keep one line of space above and below your card -->
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
 
+        <!-- ________ Sappenfield's Contributor card START ________  -->
+        <div class="card">
+          <p class="name">Albert J. Sappendield</p>
+          <p class="contact">
+            <i class="fab fa-x-twitter"></i>
+            <a href="https://twitter.com/aj_sappenfield" target="_blank">aj_sappenfield</a>
+            <i class="fab fa-linkedin"></i>
+            <a href="http://www.linkedin.com/in/a-j-sappenfield/" target="_blank">AJ Sappenfield</a>
+            <i class="fab fa-github"></i>
+            <a href="https://www.github.com" target="_blank">sappenfield</a>
```

Comment:
Thank you for your contribution! Before I merge your pull request, there's just one little thing to fix. The GitHub link you gave is to the main GitHub page, as opposed to your specific GitHub account. Once you fix that, I'll merge your request.

Reply:
Has been fixed! Thanks for pointing that out!